app/controllers/artists/registrations_controller.rb

```diff
@@ -64,7 +64,7 @@ def create
   # end
 
   # The path used after sign up for inactive accounts.
-  # def after_inactive_sign_up_path_for(resource)
-  #   super(resource)
-  # end
+  def after_inactive_sign_up_path_for(resource)
+    '/artists/confirm_email'
```

Comment:
You should use rails path helpers! It's like giving a nickname to your routes so in case the actual route changes, you don't have to go back and change your code ever. 

See: https://guides.rubyonrails.org/routing.html#naming-routes

In short: add `,  as: 'artist_confirm_email'` or something similar to the end of the route in the routes file. Then return `artist_confirm_email_path` (a function Rails creates for you to easily refer to that route!)